client/src/test.ts

```diff
@@ -7,13 +7,6 @@ import {
   platformBrowserDynamicTesting
 } from '@angular/platform-browser-dynamic/testing';
 
-declare const require: {
```

Comment:
Why don't we need this anymore? Is the default (somewhere) now to pick up all `.spec` files?